app/service/task_status_service.rb

```diff
@@ -3,9 +3,9 @@ def initialize(form:)
     @form = form
   end
 
-  def status_counts
-    { completed: all_task_status.compact.count(:completed),
-      total: all_task_status.compact.count }
+  def status_counts(current_user)
+    { completed: all_task_status(current_user).compact.count(:completed),
+      total: all_task_status(current_user).compact.count }
```

Comment:
Can we pull `all_task_status` out to avoid running it twice?

```suggestion
    all_task_status_compact = all_task_status(current_user).compact
    { completed: all_task_status_compact.count(:completed),
      total: all_task_status_compact.count }
```